include/lemlib/devices/encoder/encoder.hpp

```diff
@@ -18,15 +19,15 @@ class Encoder {
          *
          * @return float angle rotated by the encoder, in radians
```

Comment:
```suggestion
         * @return Angle angle rotated by the encoder
```
update doxygen comment